src/ShortidType.php

```diff
@@ -18,6 +20,9 @@ final class ShortidType extends Type
 {
     public const NAME = 'shortid';
 
+    /**
```

Comment:
This phpdoc is also useless to me

Reply:
I think this helps to see at the first glance that these are overloaded methods. But again, no debate on principles here, I'll remove it.